### ISSUE TITLE
Sorting of Songs based on albums / other metadata like genre

### ISSUE THREAD

**Issue by nots1dd** [cococry/lyssa#12]
First off, great application I am loving it! Since runtime dependencies include ffmpeg and jq, I figured the feature of sorting of songs can be done based on artist, album and genre, as such metadata can and is already being parsed in your application.

I am already implementing a similar type of song cache method that stores songs based on aritst > album > disc > track in my TUI music player (with very similar dependencies) so implementing this feature should not be too much of a hassle.

This type of metadata extraction can be made in a modular way, so it can be used everywhere within the application and give the user more cool features like lyrics viewing (in metadata)

**Comment by cococry** [comment]
yeah i know, i was wanting to implement this feature for a while already
but the project in paused currently so there will not be any upadtes in
like the next couple of weeks.
